filesystem.go

```diff
@@ -152,11 +152,6 @@ func (fs *filesystem) createAllResources(pod Pod) error {
 }
 
 func (fs *filesystem) storeFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
yes, that would be safest approach, as the move is atomic so you can't end up with a partially written file in the final destination.